docs/sources/panels-visualizations/query-transform-data/transform-data/index.md

```diff
@@ -399,6 +466,13 @@ I applied a transformation to join the query results using the time field. Now I
 
 {{< figure src="/static/img/docs/transformations/join-fields-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
 
+### Join by Labels
+
+Use this transformation to join multiple results into a single table. This is especially useful for converting multiple
+time series results into a single wide table with a shared Label field.
+
+Select the label to `Join` by between the labels available or common accross all time series, and the `Value` name for the output result. 
```

Comment:
When you say "the labels available or common across all time series" - are these the same thing or two different sets of labels?
```suggestion

- **Join** - Select the label to join by between the labels available or common across all time series.
- **Value** - The name for the output result. 
```